horovod/tensorflow/mpi_ops_test.py

```diff
@@ -474,12 +489,18 @@ def test_horovod_broadcast(self):
 
         with self.test_session() as session:
             dtypes = [tf.uint8, tf.int8, tf.uint16, tf.int16,
-                      tf.int32, tf.int64, tf.float32, tf.float64]
+                      tf.int32, tf.int64, tf.float32, tf.float64,
+                      tf.bool]
             dims = [1, 2, 3]
             root_ranks = list(range(size))
             for dtype, dim, root_rank in itertools.product(dtypes, dims, root_ranks):
-                tensor = tf.ones([17] * dim, dtype=dtype) * rank
-                root_tensor = tf.ones([17] * dim, dtype=dtype) * root_rank
+                tensor = tf.ones([17] * dim)
+                root_tensor = tf.ones([17] * dim)
```

Comment:
For above two lines, need to multiply by rank

Reply:
Good catch!